rules/ensure-default-service-accounts-has-only-default-roles/test/failed_rolebinding/input/rolebinding.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name:  pod
+  namespace: default
+subjects:
+- kind: User  
+  name: jane 
+  apiGroup: rbac.authorization.k8s.io
+- kind: ServiceAccount
+  name: john-dev
+  namespace: default
```

Comment:
why does this rolebinding fail? It has no SA named default in its subjects

Reply:
fixed